frontend/src/concepts/pipelines/content/utils.tsx

```diff
@@ -107,3 +109,36 @@ export const isPipelineRun = (resource: PipelineCoreResourceKF): resource is Pip
 export const isPipelineRecurringRun = (
   resource: PipelineCoreResourceKF,
 ): resource is PipelineRecurringRunKF => 'recurring_run_id' in resource && !('run_id' in resource);
+
+// workaround until https://issues.redhat.com/browse/RHOAIENG-27727 is fixed
+// after the above ticket is resolved, once 'failingTodeploy' is detected then it is an automatic fail
+// without any need for a timeout.
+// but we should still timeout everything else
```

Comment:
can you create a followup jira that is blocked by https://issues.redhat.com/browse/RHOAIENG-27727. so we can track it

Reply:
added the followup ticket into the comment as well